Home.css

```diff
@@ -0,0 +1,108 @@
+@import url("color-variable.css");
+@import url("Navigation.css");
+
+/* css for home page */
+.outer-container{
+    height: 35.8rem;
+    width: 81rem;
+    overflow-y: scroll;
+}
+.headings{
+    text-align: center;
+}
+h1{
+    color: var(--white-color);
+}
+h2{
+    color: var(--white-color);
+}
+.first-section{
+    height: 18rem;
+    width: 70rem;
+    text-align: center;
+    margin-top: 4rem;
+    display: flex;
+    flex-wrap: wrap;
+    gap: 2rem;
+    margin-left: 9rem;
+}
+.categories:hover{
+    transform: scale(1.1);
+}
+.cricket-box{
+    height: 14rem;
+    width: 19rem;
+    border-radius: 2rem;
+    background: var(--cricket-bg-color)
+}
+.paragraph{
+    height: 2rem;
+    width: 14rem;
+    margin-top: -0.5rem;
+    margin-left: 3rem;
+    font-size: 1rem;
+    color: var(--white-color);
+}
+.button{
+    margin-top: -0.8rem;
+    color: var(--white-color);
+}
+.fa-unlock{
+    position: relative;
+    top: -14rem;
+    margin-left: -14rem;
+    color:var(--white-color);
+    cursor: pointer;
+    font-size:3rem;
+    
+}
+.fa-lock{
```

Comment:
I can't understand the class naming what fa-lock and unlock means here

Reply:
These are the icon's classess .

Reply:
Ohh ok,Fine then